addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -1,46 +0,0 @@
-odoo.define("website.tour.edit_megamenu", function (require) {
-"use strict";
-
-const tour = require('web_tour.tour');
-
-tour.register('edit_megamenu', {
```

Comment:
Is not this the test without python test I commented in your other PR? Why is it being removed in a stable version which could require it? Also if your other test expands this one, should not this be kept anyway?

Reply:
I wanted to move the existing test on the mega menu, that was not played at install, to a new, more global test on the menus with a few more actions on the regular menus. Should I keep this one as it is (and add it from the python test suite), and add a new tour 'edit_menu' ?

Reply:
Yes, and in master you could remove the duplicated test.